packages/synapse-sdk/src/test/sp-registry-service.test.ts

```diff
@@ -434,22 +413,19 @@ describe('SPRegistryService', () => {
       // Override to return provider without products
       ;(service as any)._getRegistryContract = () => ({
         ...createMockContract(),
-        getProviderWithProduct: async () => ({
-          providerId: 1,
-          providerInfo: {
-            id: BigInt(1),
-            serviceProvider: mockProviderAddress,
-            payee: mockProviderAddress,
-            name: 'Test Provider',
-            description: 'A test storage provider',
-            isActive: true,
-          },
-          product: {
-            productType: 0n,
-            capabilityKeys: [],
-            isActive: false,
+        getPDPService: async () => ({
```

Comment:
This is wrong. There is no getPDPService method! Why did you revert my fix that switched to the new method `getProviderWithProduct`?